### DISCUSSION TITLE
Beep when start TX

### DISCUSSION THREAD

**Discussion by karacsonyisandor** [egzumer/uv-k5-firmware-custom#70] [Ideas]
Include an option for a BEEP when start transmitting. Right after PTT is pressed and TX is active. Also, an ERROR tone when Busy Channel  Lockout is enabled. It could be an option inside the BEEP menu like PTT / Menu / Both / Off

**Comment by ErikS-web** [comment]
A `Roger`-beep at the _end of transmission_ (OFF/ROGER/MDC1200) is under Under Menu 41. 

What is the use to have it at the start/ beginning of transmission ?

Your second suggestion ... (better made it a _separate item_).

Did you try an external microphone ? The PTT act like the `41. BusyCL` (busy channel lockout). See [`external-key-PTT`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#external-key)
That other functionality might be useful ;-)

**Reply by karacsonyisandor** [reply]
I am not talking about the end roger beep :)
We use DMR radios on daily basis, all our radios have a short beep on the radio when you push PTT and the radio connects to the repeater. This way you know you can talk and the TX is ok. There is also an error beep when out of range or channel is busy. All the beeps are on the local radio and not transmitted (the repeater sends a short roger beep at the end). 

On this Quansheng analog radio the beep would confirm that you are actually transmitting and not just pushing the PTT without TX being active - for any reason... 

For those who use radios on daily basis this is a useful function. We had an event with about 120 analog radios on 2 channels without any kind of roger beep or BCL beep and it was a chaos...

**Reply by ydms** [reply]
Settings menu I know it's there
I have also set it
No matter what option I set
He always has no voice

**Reply by ErikS-web** [reply]
I think this is posted wrong. And can be deleted. See https://github.com/egzumer/uv-k5-firmware-custom/discussions/83

**Comment by qqk8** [comment]
I seem to have the same problem after upgrading to 16.

**Reply by egzumer** [reply]
What problem? Nothing is changed in 16 regarding any of that.

**Reply by ErikS-web** [reply]
I thinks this is reaction on wrong posting of user YDMS. Suggest to delete

**Comment by ErikS-web** [comment]
UV-K5 is an Analog radio and has no DMR -functionality. 

Radio amateurs use a unieke roepletters ( call ) as identifier, and no annoying beeps.
A roger-beep is only used during contest.

For good communication, ether discipline between users is always necessary.

**Reply by karacsonyisandor** [reply]
**I do not think you understand what I have written...** 

I know UV K5 is analog, I am also a radio amateur and **the start TX local beep is a nice option to have**. 
Also do not compare amateur radio communication with event and emergency communications... 
I have about 25 years' experience in both so I know what I am talking about.

**Comment by egzumer** [comment]
Sounding a tone without muting incoming transmission is not possible as far as I know, so this rules out TX error beep on lockout I would say.

**Comment by ErikS-web** [comment]
- Do you see the led indicating RED on TX (PTT pressed)? There is also a TX on screen, in front of selected VFO.

- And with Busy Channel Lockout enabled,  and a signal is received the led is GREEN ( also a RX on screen). Than the led wil not become RED when PTT is pressed and still recieving a signal, but received audio is than muted.

**Comment by Arikshten** [comment]
This may be helped by using the existing option for PTT Apollo Quindar. 
Menu option 48  (in version 0.19) - 
This adds the Apollo Quindar tones to when you start transmitting and stop. While not exactly the same as you are asking for it might be of help. 


Hope that is of some help.
